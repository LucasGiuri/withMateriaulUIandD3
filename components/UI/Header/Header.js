import Image from '../Image/Image';
import Menu from '../../containers/Menu';
import { HeaderContainer } from './Header.styles';
import NavLink from '../NavLink/NavLink';

const Header = () => (
  <HeaderContainer>
    <NavLink href="/" as="/">
      <Image isLogo={true} url="/static/icons/logo.png" alt="logo" />
    </NavLink>
    <Menu />
  </HeaderContainer>
);

export default Header;