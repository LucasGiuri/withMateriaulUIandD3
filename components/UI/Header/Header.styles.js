import styled from 'styled-components';
import { WHITE } from '../../../public/styles/colors';

export const HeaderContainer = styled.header`
  display: flex;
  align-items: center;
  flex-direction: column;
  width: 100%;
  background-color: ${WHITE}
`;

export const Nav = styled.header`
  display: flex;
  width: 100%;
  justify-content: space-around;
  align-items: center;
`;