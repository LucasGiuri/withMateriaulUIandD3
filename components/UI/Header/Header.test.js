import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';
import 'jest-styled-components';

jest.mock('../../containers/Menu', () => 'Menu');

describe('Header', () => {
  it('Header basic render snapshoot', () => {
    const component = renderer.create(<Header />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
