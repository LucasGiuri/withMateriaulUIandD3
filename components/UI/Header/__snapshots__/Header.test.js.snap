// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header basic render snapshoot 1`] = `
.c2 {
  padding-top: 8px;
  padding-bottom: 8px;
  width: 100px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  background-color: #FFFFFF;
}

.c1 {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-family: 'Roboto',sans-serif;
}

<header
  className="c0"
>
  <a
    className="c1"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <img
      alt="logo"
      className="c2"
      onClick={[Function]}
      src="/static/icons/logo.png"
    />
  </a>
  <Menu />
</header>
`;
